recipes/flexx/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "4.0.0rc1" %}
+
+package:
+  name: example
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/zoofIO/flexx.git
+  git_tag: b824bf9f9ab7d441b492d42484bb88877cd4af93
+
+build:
+  script: python setup.py install
+  number: 0
+
+requirements:
+  build:
+    - python
+    - tornado
+  run:
+    - python
+    - tornado
+
+test:
+  imports:
+    - flexx.ui
+    - flexx.app
+    - flexx.event
+    - flexx.pyscript
+    - flexx.webruntime
+    - flexx.util
+
+about:
+  home: http://github.com/zoofIO/flexx
+  doc_url: http://flexx.readthedocs.io
+  dev_url: http://github.com/zoofIO/flexx
+  summary: Python UI tookit based on web technology
+  license: 2-clause BSD
```

Comment:
We are using `BSD 2-Clause` instead. That is not mandatory, but will make the lives of automated tools much easier when researching the packages licenses in the recipes.